backend/app.js

```diff
@@ -0,0 +1,8 @@
+var express = require('express');
+var app = express();
+app.get('/', function (req, res) {
+  res.send('Hello World!');
+});
+app.listen(3000, function () {
```

Comment:
Arrow functions is less error prone:
```suggestion
app.listen(3000, () => {
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions